integration_test/functions/src/v1/testLab-utils.ts

```diff
@@ -1,5 +1,5 @@
-import fetch from 'node-fetch';
 import * as admin from 'firebase-admin';
+import fetch from 'node-fetch';
```

Comment:
q: do you have some eslint locally that automatically formats this? (or is it something you did manually)

Reply:
npm run format:fix && npm run lint:fix